freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientExceptionUtil.java

```diff
@@ -57,8 +57,6 @@
             HttpStatus.REQUEST_TIMEOUT,
             HttpStatus.TOO_MANY_REQUESTS,
             HttpStatus.INTERNAL_SERVER_ERROR,
-            HttpStatus.BAD_GATEWAY,
-            HttpStatus.SERVICE_UNAVAILABLE,
```

Comment:
Wouldnt these be the cases we do want to retry on?  Bad Gateway means it still could be starting up.